rmw_fastrtps_shared_cpp/src/rmw_node.cpp

```diff
@@ -214,6 +214,40 @@ get_security_file_paths(
   return true;
 }
 
+bool
+get_security_logging_file_path(
```

Comment:
I have mixed feelings about this being a separate function given the duplication, but I understand why it was done for the separation of concerns. I think you can vastly improve things by writing a third function that includes the duplicated logic, and use it from both.

Reply:
Let me improve the code duplication situation :+1:

Reply:
Done.